CHANGELOG.txt

```diff
@@ -1,3 +1,134 @@
+Airflow 1.10.6, 2019-10-21
+--------------------------
+
+New Feature
+"""""""""""
+- [AIRFLOW-4908] Implement BigQuery Hooks/Operators for update_dataset, patch_dataset and get_dataset (#5546)
+- [AIRFLOW-4741] Optionally report task errors to Sentry (#5407)
+- [AIRFLOW-4939] Add default_task_retries config (#5570)
+- [AIRFLOW-5508] Add config setting to limit which StatsD metrics are emitted (#6130)
+- [AIRFLOW-4222] Add cli autocomplete for bash & zsh (#5789)
+- [AIRFLOW-3871] Operators template fields can now render fields inside objects (#4743)
+
+Improvement
```

Comment:
```suggestion
Improvements
```

Reply:
Need to update my automated generator :)